src/components/TextFmt.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { Text } from 'react-native';
+import { Strong } from '../utils';
+
+const TextFmt = (props) => {
```

Comment:
Would be good to have a comment explaining this component.

Reply:
Done. Thanks!